openassessment/xblock/test/test_submission.py

```diff
@@ -964,7 +963,7 @@ def test_open_saved_response_misaligned_file_data(self, xblock):
         self._assert_path_and_context(
             xblock, 'openassessmentblock/response/oa_response.html',
             {
-                'text_response': 'required',
+                'text_response': None,
```

Comment:
This changed because I switched the scenario, needed to run the `parse_xml` import code.